### PR TITLE
[24.0] Fix Workflow Name Display in `WorkflowCard` to Prevent Bleeding Over Borders

### LINKED ISSUE #17712: workflow grid card contents can bleed over the borders
maybe because of a combination of long name and only two items?

Screenshot 2024-03-13 at 7 22 22 PM

**Galaxy Version and/or server at which you observed the bug**
Galaxy Version: 24.0

**Browser and Operating System**
Operating System: macOS
Browser:, Chrome,

### PR DESCRIPTION
**Pull request by itisAliRH**
Fixes #17712

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

**Review by jmchilton — APPROVED** [review]
Tool shed tests look unrelated. I'll merge this after the Selenium tests finish. Thanks for the fix.

File: client/src/components/Workflow/WorkflowCard.vue [hunk 1]
```diff
@@ -329,19 +331,7 @@ async function onTagClick(tag: string) {
         }
     }
 
-    .workflow-shared {
-        border-left: 0.25rem solid $brand-primary;
-    }
-
     @container workflow-card (max-width: #{$breakpoint-md}) {
-        .hide-in-card {
-            display: none !important;
```
Comment (martenson) [inline client/src/components/Workflow/WorkflowCard.vue:338]: so many `!important`s disappearing, me like!

Comment (itisAliRH) [reply client/src/components/Workflow/WorkflowCard.vue:338]: These classes are not used anymore!

**Comment by itisAliRH** [comment]
@martenson Fixed! Thanks :)

File: client/src/components/Workflow/WorkflowCard.vue (hunk 1 above)
Comment (martenson) [reply client/src/components/Workflow/WorkflowCard.vue:338]: yeah, it's great

**Comment by martenson** [comment]
thanks @itisAliRH !

**Pull request merged**